lib/state.js

```diff
@@ -157,7 +164,7 @@ StateResolver.prototype.capture_ = function() {
                                     result.error, true)
         };
       } else {
-        evaluated = that.resolveVariable_(expression, result.mirror);
+        evaluated = that.resolveVariable_(expression, result.mirror, true);
```

Comment:
This has been fixed to remove the duplicated information without modifying the V8 object.